javaslang/src/main/java/javaslang/collection/Array.java

```diff
@@ -1059,6 +1059,10 @@ public int length() {
         return Collections.scanRight(this, zero, operation, Array::ofAll);
     }
 
+    @Override public Array<T> shuffle() {
```

Comment:
minor: please reformat these, as you can see, in other places the @`Override` is on a separate line